package.json

```diff
@@ -17,6 +17,7 @@
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^5.40.0",
     "@typescript-eslint/parser": "^5.40.0",
+    "axe-test-fixtures": "github:dequelabs/axe-test-fixtures#v1",
```

Comment:
This wasn't pinned to `v1` so wasn't getting the path update. Also not sure why it was a dependency.